2013/08/01 17:18:41 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not allowed to access this page.' in /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/GiiModule.php:168
Stack trace:
#0 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(263): GiiModule->beforeControllerAction(Object(DefaultController), Object(CInlineAction))
#1 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#2 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii')
#3 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#4 /home/work/wangming/comse/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=gii
---
2013/08/01 17:19:11 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not allowed to access this page.' in /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/GiiModule.php:168
Stack trace:
#0 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(263): GiiModule->beforeControllerAction(Object(DefaultController), Object(CInlineAction))
#1 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#2 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii')
#3 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#4 /home/work/wangming/comse/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=gii
---
2013/08/01 17:23:46 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Can't connect to local MySQL server through socket '/home/work/local/wangming/mysql5/var/mysql.sock' (2)' in /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/work/wangming/comse/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://10.46.168.18:8888/index.php?r=gii/default/index
---
2013/08/01 17:24:17 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Can't connect to local MySQL server through socket '/home/work/local/wangming/mysql5/var/mysql.sock' (2)' in /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/work/wangming/comse/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://10.46.168.18:8888/index.php?r=gii/default/index
---
2013/08/01 17:25:41 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'cq01-pl39.cq01.baidu.com' (using password: YES)' in /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/work/wangming/comse/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://10.46.168.18:8888/index.php?r=gii/default/index
---
2013/08/01 18:04:23 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Can't connect to local MySQL server through socket '/home/work/local/wangming/mysql5/var/mysql.sock' (2)' in /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/work/wangming/comse/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://10.46.168.18:8888/index.php?r=gii/default/index
---
2013/08/01 18:13:29 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Can't connect to local MySQL server through socket '/home/work/local/wangming/mysql5/var/mysql.sock' (2)' in /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/work/wangming/comse/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://10.46.168.18:8888/index.php?r=gii/default/index
---
2013/08/01 18:13:31 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Can't connect to local MySQL server through socket '/home/work/local/wangming/mysql5/var/mysql.sock' (2)' in /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/work/wangming/comse/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://10.46.168.18:8888/index.php?r=gii/default/index
---
2013/08/01 18:16:01 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Can't connect to local MySQL server through socket '/home/work/local/wangming/mysql5/var/mysql.sock' (2)' in /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/work/wangming/comse/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://10.46.168.18:8888/index.php?r=gii/default/index
---
2013/08/01 18:16:04 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Can't connect to local MySQL server through socket '/home/work/local/wangming/mysql5/var/mysql.sock' (2)' in /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/work/wangming/comse/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://10.46.168.18:8888/index.php?r=gii/default/index
---
2013/08/01 18:24:24 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Can't connect to local MySQL server through socket '/home/work/local/wangming/mysql5/var/mysql.sock' (2)' in /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/work/wangming/comse/index.php(14): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
HTTP_REFERER=http://10.46.168.18:8888/index.php?r=gii
---
2013/08/01 18:25:16 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Can't connect to local MySQL server through socket '/home/work/local/wangming/mysql5/var/mysql.sock' (2)' in /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/work/wangming/comse/index.php(14): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
---
2013/08/01 18:26:48 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Can't connect to local MySQL server through socket '/home/work/local/wangming/mysql5/var/mysql.sock' (2)' in /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/work/wangming/comse/index.php(14): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
---
2013/08/01 18:26:51 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Can't connect to local MySQL server through socket '/home/work/local/wangming/mysql5/var/mysql.sock' (2)' in /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/work/wangming/comse/index.php(14): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
---
2013/08/01 18:26:52 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Can't connect to local MySQL server through socket '/home/work/local/wangming/mysql5/var/mysql.sock' (2)' in /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/work/wangming/comse/index.php(14): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
---
2013/08/01 18:26:52 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Can't connect to local MySQL server through socket '/home/work/local/wangming/mysql5/var/mysql.sock' (2)' in /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/work/wangming/comse/index.php(14): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
---
2013/08/01 18:26:52 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Can't connect to local MySQL server through socket '/home/work/local/wangming/mysql5/var/mysql.sock' (2)' in /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/work/wangming/comse/index.php(14): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
---
2013/08/01 18:26:52 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Can't connect to local MySQL server through socket '/home/work/local/wangming/mysql5/var/mysql.sock' (2)' in /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php:381
Stack trace:
#0 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(330): CDbConnection->open()
#1 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/db/CDbConnection.php(308): CDbConnection->setActive(true)
#2 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CModule.php(387): CDbConnection->init()
#3 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CModule.php(103): CModule->getComponent('db')
#4 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/generators/model/ModelCode.php(57): CModule->__get('db')
#5 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CFormModel.php(39): ModelCode->init()
#6 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(153): CFormModel->__construct()
#7 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/gii/CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#10 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#11 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#13 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii/model')
#14 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /home/work/wangming/comse/index.php(14): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=gii/model
---
2013/08/01 18:34:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message '无法解析请求 "syncinstance"。' in /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php:286
Stack trace:
#0 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('syncinstance')
#1 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#2 /home/work/wangming/comse/index.php(14): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=syncinstance
---
2013/08/01 19:02:34 [error] [php] YiiBase::include(CRadioButtonColumn.php) [<a href='function.YiiBase-include'>function.YiiBase-include</a>]: failed to open stream: No such file or directory (/home/work/wangming/local/yii-1.1.13.e9e4a0/framework/YiiBase.php:421)
Stack trace:
#0 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/YiiBase.php(196): import()
#1 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/zii/widgets/grid/CGridView.php(363): createComponent()
#2 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/zii/widgets/grid/CGridView.php(334): CGridView->initColumns()
#3 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CBaseController.php(147): CGridView->init()
#4 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CBaseController.php(172): SyncInstanceController->createWidget()
#5 /home/work/wangming/comse/protected/views/syncInstance/admin.php(83): SyncInstanceController->widget()
#6 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CBaseController.php(126): require()
#7 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CBaseController.php(95): SyncInstanceController->renderInternal()
#8 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(869): SyncInstanceController->renderFile()
#9 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(782): SyncInstanceController->renderPartial()
#10 /home/work/wangming/comse/protected/controllers/SyncInstanceController.php(143): SyncInstanceController->render()
#11 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): SyncInstanceController->actionAdmin()
#12 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams()
#13 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/filters/CFilterChain.php(133): SyncInstanceController->runAction()
#14 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/filters/CFilter.php(40): CFilterChain->run()
#15 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(1145): CAccessControlFilter->filter()
#16 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/filters/CInlineFilter.php(58): SyncInstanceController->filterAccessControl()
#17 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter()
#18 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(291): CFilterChain->run()
#19 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(265): SyncInstanceController->runActionWithFilters()
#20 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): SyncInstanceController->run()
#21 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController()
#22 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#23 /home/work/wangming/comse/index.php(14): CWebApplication->run()
REQUEST_URI=/index.php?r=syncInstance/admin
in /home/work/wangming/comse/protected/views/syncInstance/admin.php (83)
in /home/work/wangming/comse/protected/controllers/SyncInstanceController.php (143)
in /home/work/wangming/comse/index.php (14)
2013/08/01 19:03:13 [error] [php] YiiBase::include(CRadioButtonColumn.php) [<a href='function.YiiBase-include'>function.YiiBase-include</a>]: failed to open stream: No such file or directory (/home/work/wangming/local/yii-1.1.13.e9e4a0/framework/YiiBase.php:421)
Stack trace:
#0 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/YiiBase.php(196): import()
#1 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/zii/widgets/grid/CGridView.php(363): createComponent()
#2 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/zii/widgets/grid/CGridView.php(334): CGridView->initColumns()
#3 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CBaseController.php(147): CGridView->init()
#4 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CBaseController.php(172): SyncInstanceController->createWidget()
#5 /home/work/wangming/comse/protected/views/syncInstance/admin.php(83): SyncInstanceController->widget()
#6 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CBaseController.php(126): require()
#7 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CBaseController.php(95): SyncInstanceController->renderInternal()
#8 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(869): SyncInstanceController->renderFile()
#9 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(782): SyncInstanceController->renderPartial()
#10 /home/work/wangming/comse/protected/controllers/SyncInstanceController.php(143): SyncInstanceController->render()
#11 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): SyncInstanceController->actionAdmin()
#12 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams()
#13 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/filters/CFilterChain.php(133): SyncInstanceController->runAction()
#14 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/filters/CFilter.php(40): CFilterChain->run()
#15 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(1145): CAccessControlFilter->filter()
#16 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/filters/CInlineFilter.php(58): SyncInstanceController->filterAccessControl()
#17 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter()
#18 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(291): CFilterChain->run()
#19 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CController.php(265): SyncInstanceController->runActionWithFilters()
#20 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): SyncInstanceController->run()
#21 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController()
#22 /home/work/wangming/local/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#23 /home/work/wangming/comse/index.php(14): CWebApplication->run()
REQUEST_URI=/index.php?r=syncInstance/admin
in /home/work/wangming/comse/protected/views/syncInstance/admin.php (83)
in /home/work/wangming/comse/protected/controllers/SyncInstanceController.php (143)
in /home/work/wangming/comse/index.php (14)
2013/12/14 15:34:05 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not allowed to access this page.' in /var/www/yii-1.1.13.e9e4a0/framework/gii/GiiModule.php:168
Stack trace:
#0 /var/www/yii-1.1.13.e9e4a0/framework/web/CController.php(263): GiiModule->beforeControllerAction(Object(DefaultController), Object(CInlineAction))
#1 /var/www/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('')
#2 /var/www/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('gii')
#3 /var/www/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#4 /var/www/webroot/meilizhubo/index.php(14): CApplication->run()
#5 {main}
REQUEST_URI=/index.php?r=gii
---
2013/12/14 15:39:57 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[42000]: Syntax error or access violation: 1142 INSERT command denied to user 'test'@'localhost' for table 'zhubo'. The SQL statement executed was: INSERT INTO `zhubo` (`local_id`, `url`, `name`, `head_img`, `site_id`, `level`, `wealth_level`, `time_level`, `sex`, `region`, `familys`, `constellation`, `age`, `hots`, `fans`, `tags`, `news_num`, `news_photo_num`, `is_live`, `last_live_time`, `photos`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8, :yp9, :yp10, :yp11, :yp12, :yp13, :yp14, :yp15, :yp16, :yp17, :yp18, :yp19, :yp20).
2013/12/14 15:39:57 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand 无法执行 SQL 语句: SQLSTATE[42000]: Syntax error or access violation: 1142 INSERT command denied to user 'test'@'localhost' for table 'zhubo'' in /var/www/yii-1.1.13.e9e4a0/framework/db/CDbCommand.php:357
Stack trace:
#0 /var/www/yii-1.1.13.e9e4a0/framework/db/ar/CActiveRecord.php(1023): CDbCommand->execute()
#1 /var/www/yii-1.1.13.e9e4a0/framework/db/ar/CActiveRecord.php(795): CActiveRecord->insert(NULL)
#2 /var/www/webroot/meilizhubo/protected/controllers/ZhuboController.php(73): CActiveRecord->save()
#3 /var/www/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): ZhuboController->actionCreate()
#4 /var/www/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /var/www/yii-1.1.13.e9e4a0/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 /var/www/yii-1.1.13.e9e4a0/framework/web/filters/CFilter.php(40): CFilterChain->run()
#7 /var/www/yii-1.1.13.e9e4a0/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 /var/www/yii-1.1.13.e9e4a0/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 /var/www/yii-1.1.13.e9e4a0/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 /var/www/yii-1.1.13.e9e4a0/framework/web/CController.php(291): CFilterChain->run()
#11 /var/www/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('create')
#13 /var/www/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('zhubo/create')
#14 /var/www/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /var/www/webroot/meilizhubo/index.php(14): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=zhubo/create
HTTP_REFERER=http://192.168.216.128:8888/index.php?r=zhubo/create
---
2013/12/14 15:40:29 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[42000]: Syntax error or access violation: 1142 INSERT command denied to user 'test'@'localhost' for table 'zhubo'. The SQL statement executed was: INSERT INTO `zhubo` (`local_id`, `url`, `name`, `head_img`, `site_id`, `level`, `wealth_level`, `time_level`, `sex`, `region`, `familys`, `constellation`, `age`, `hots`, `fans`, `tags`, `news_num`, `news_photo_num`, `is_live`, `last_live_time`, `photos`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8, :yp9, :yp10, :yp11, :yp12, :yp13, :yp14, :yp15, :yp16, :yp17, :yp18, :yp19, :yp20).
2013/12/14 15:40:29 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand 无法执行 SQL 语句: SQLSTATE[42000]: Syntax error or access violation: 1142 INSERT command denied to user 'test'@'localhost' for table 'zhubo'' in /var/www/yii-1.1.13.e9e4a0/framework/db/CDbCommand.php:357
Stack trace:
#0 /var/www/yii-1.1.13.e9e4a0/framework/db/ar/CActiveRecord.php(1023): CDbCommand->execute()
#1 /var/www/yii-1.1.13.e9e4a0/framework/db/ar/CActiveRecord.php(795): CActiveRecord->insert(NULL)
#2 /var/www/webroot/meilizhubo/protected/controllers/ZhuboController.php(73): CActiveRecord->save()
#3 /var/www/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): ZhuboController->actionCreate()
#4 /var/www/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /var/www/yii-1.1.13.e9e4a0/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 /var/www/yii-1.1.13.e9e4a0/framework/web/filters/CFilter.php(40): CFilterChain->run()
#7 /var/www/yii-1.1.13.e9e4a0/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 /var/www/yii-1.1.13.e9e4a0/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 /var/www/yii-1.1.13.e9e4a0/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 /var/www/yii-1.1.13.e9e4a0/framework/web/CController.php(291): CFilterChain->run()
#11 /var/www/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('create')
#13 /var/www/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('zhubo/create')
#14 /var/www/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /var/www/webroot/meilizhubo/index.php(14): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=zhubo/create
HTTP_REFERER=http://192.168.216.128:8888/index.php?r=zhubo/create
---
2013/12/14 15:40:33 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[42000]: Syntax error or access violation: 1142 INSERT command denied to user 'test'@'localhost' for table 'zhubo'. The SQL statement executed was: INSERT INTO `zhubo` (`local_id`, `url`, `name`, `head_img`, `site_id`, `level`, `wealth_level`, `time_level`, `sex`, `region`, `familys`, `constellation`, `age`, `hots`, `fans`, `tags`, `news_num`, `news_photo_num`, `is_live`, `last_live_time`, `photos`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8, :yp9, :yp10, :yp11, :yp12, :yp13, :yp14, :yp15, :yp16, :yp17, :yp18, :yp19, :yp20).
2013/12/14 15:40:33 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand 无法执行 SQL 语句: SQLSTATE[42000]: Syntax error or access violation: 1142 INSERT command denied to user 'test'@'localhost' for table 'zhubo'' in /var/www/yii-1.1.13.e9e4a0/framework/db/CDbCommand.php:357
Stack trace:
#0 /var/www/yii-1.1.13.e9e4a0/framework/db/ar/CActiveRecord.php(1023): CDbCommand->execute()
#1 /var/www/yii-1.1.13.e9e4a0/framework/db/ar/CActiveRecord.php(795): CActiveRecord->insert(NULL)
#2 /var/www/webroot/meilizhubo/protected/controllers/ZhuboController.php(73): CActiveRecord->save()
#3 /var/www/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): ZhuboController->actionCreate()
#4 /var/www/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /var/www/yii-1.1.13.e9e4a0/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 /var/www/yii-1.1.13.e9e4a0/framework/web/filters/CFilter.php(40): CFilterChain->run()
#7 /var/www/yii-1.1.13.e9e4a0/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 /var/www/yii-1.1.13.e9e4a0/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 /var/www/yii-1.1.13.e9e4a0/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 /var/www/yii-1.1.13.e9e4a0/framework/web/CController.php(291): CFilterChain->run()
#11 /var/www/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('create')
#13 /var/www/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('zhubo/create')
#14 /var/www/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /var/www/webroot/meilizhubo/index.php(14): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=zhubo/create
HTTP_REFERER=http://192.168.216.128:8888/index.php?r=zhubo/create
---
2013/12/14 15:40:34 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[42000]: Syntax error or access violation: 1142 INSERT command denied to user 'test'@'localhost' for table 'zhubo'. The SQL statement executed was: INSERT INTO `zhubo` (`local_id`, `url`, `name`, `head_img`, `site_id`, `level`, `wealth_level`, `time_level`, `sex`, `region`, `familys`, `constellation`, `age`, `hots`, `fans`, `tags`, `news_num`, `news_photo_num`, `is_live`, `last_live_time`, `photos`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8, :yp9, :yp10, :yp11, :yp12, :yp13, :yp14, :yp15, :yp16, :yp17, :yp18, :yp19, :yp20).
2013/12/14 15:40:34 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand 无法执行 SQL 语句: SQLSTATE[42000]: Syntax error or access violation: 1142 INSERT command denied to user 'test'@'localhost' for table 'zhubo'' in /var/www/yii-1.1.13.e9e4a0/framework/db/CDbCommand.php:357
Stack trace:
#0 /var/www/yii-1.1.13.e9e4a0/framework/db/ar/CActiveRecord.php(1023): CDbCommand->execute()
#1 /var/www/yii-1.1.13.e9e4a0/framework/db/ar/CActiveRecord.php(795): CActiveRecord->insert(NULL)
#2 /var/www/webroot/meilizhubo/protected/controllers/ZhuboController.php(73): CActiveRecord->save()
#3 /var/www/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): ZhuboController->actionCreate()
#4 /var/www/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /var/www/yii-1.1.13.e9e4a0/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 /var/www/yii-1.1.13.e9e4a0/framework/web/filters/CFilter.php(40): CFilterChain->run()
#7 /var/www/yii-1.1.13.e9e4a0/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 /var/www/yii-1.1.13.e9e4a0/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 /var/www/yii-1.1.13.e9e4a0/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 /var/www/yii-1.1.13.e9e4a0/framework/web/CController.php(291): CFilterChain->run()
#11 /var/www/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('create')
#13 /var/www/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('zhubo/create')
#14 /var/www/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /var/www/webroot/meilizhubo/index.php(14): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=zhubo/create
HTTP_REFERER=http://192.168.216.128:8888/index.php?r=zhubo/create
---
2013/12/14 15:40:36 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[42000]: Syntax error or access violation: 1142 INSERT command denied to user 'test'@'localhost' for table 'zhubo'. The SQL statement executed was: INSERT INTO `zhubo` (`local_id`, `url`, `name`, `head_img`, `site_id`, `level`, `wealth_level`, `time_level`, `sex`, `region`, `familys`, `constellation`, `age`, `hots`, `fans`, `tags`, `news_num`, `news_photo_num`, `is_live`, `last_live_time`, `photos`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8, :yp9, :yp10, :yp11, :yp12, :yp13, :yp14, :yp15, :yp16, :yp17, :yp18, :yp19, :yp20).
2013/12/14 15:40:36 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand 无法执行 SQL 语句: SQLSTATE[42000]: Syntax error or access violation: 1142 INSERT command denied to user 'test'@'localhost' for table 'zhubo'' in /var/www/yii-1.1.13.e9e4a0/framework/db/CDbCommand.php:357
Stack trace:
#0 /var/www/yii-1.1.13.e9e4a0/framework/db/ar/CActiveRecord.php(1023): CDbCommand->execute()
#1 /var/www/yii-1.1.13.e9e4a0/framework/db/ar/CActiveRecord.php(795): CActiveRecord->insert(NULL)
#2 /var/www/webroot/meilizhubo/protected/controllers/ZhuboController.php(73): CActiveRecord->save()
#3 /var/www/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): ZhuboController->actionCreate()
#4 /var/www/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /var/www/yii-1.1.13.e9e4a0/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 /var/www/yii-1.1.13.e9e4a0/framework/web/filters/CFilter.php(40): CFilterChain->run()
#7 /var/www/yii-1.1.13.e9e4a0/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 /var/www/yii-1.1.13.e9e4a0/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 /var/www/yii-1.1.13.e9e4a0/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 /var/www/yii-1.1.13.e9e4a0/framework/web/CController.php(291): CFilterChain->run()
#11 /var/www/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('create')
#13 /var/www/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('zhubo/create')
#14 /var/www/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /var/www/webroot/meilizhubo/index.php(14): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=zhubo/create
HTTP_REFERER=http://192.168.216.128:8888/index.php?r=zhubo/create
---
2013/12/14 15:41:37 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[42000]: Syntax error or access violation: 1142 INSERT command denied to user 'test'@'localhost' for table 'zhubo'. The SQL statement executed was: INSERT INTO `zhubo` (`local_id`, `url`, `name`, `head_img`, `site_id`, `level`, `wealth_level`, `time_level`, `sex`, `region`, `familys`, `constellation`, `age`, `hots`, `fans`, `tags`, `news_num`, `news_photo_num`, `is_live`, `last_live_time`, `photos`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8, :yp9, :yp10, :yp11, :yp12, :yp13, :yp14, :yp15, :yp16, :yp17, :yp18, :yp19, :yp20).
2013/12/14 15:41:37 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand 无法执行 SQL 语句: SQLSTATE[42000]: Syntax error or access violation: 1142 INSERT command denied to user 'test'@'localhost' for table 'zhubo'' in /var/www/yii-1.1.13.e9e4a0/framework/db/CDbCommand.php:357
Stack trace:
#0 /var/www/yii-1.1.13.e9e4a0/framework/db/ar/CActiveRecord.php(1023): CDbCommand->execute()
#1 /var/www/yii-1.1.13.e9e4a0/framework/db/ar/CActiveRecord.php(795): CActiveRecord->insert(NULL)
#2 /var/www/webroot/meilizhubo/protected/controllers/ZhuboController.php(73): CActiveRecord->save()
#3 /var/www/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(49): ZhuboController->actionCreate()
#4 /var/www/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /var/www/yii-1.1.13.e9e4a0/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 /var/www/yii-1.1.13.e9e4a0/framework/web/filters/CFilter.php(40): CFilterChain->run()
#7 /var/www/yii-1.1.13.e9e4a0/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 /var/www/yii-1.1.13.e9e4a0/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 /var/www/yii-1.1.13.e9e4a0/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 /var/www/yii-1.1.13.e9e4a0/framework/web/CController.php(291): CFilterChain->run()
#11 /var/www/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('create')
#13 /var/www/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('zhubo/create')
#14 /var/www/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /var/www/webroot/meilizhubo/index.php(14): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=zhubo/create
HTTP_REFERER=http://192.168.216.128:8888/index.php?r=zhubo/create
---
2013/12/14 16:58:15 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in /var/www/webroot/meilizhubo/protected/controllers/ZhuboController.php:157
Stack trace:
#0 /var/www/webroot/meilizhubo/protected/controllers/ZhuboController.php(55): ZhuboController->loadModel('1')
#1 [internal function]: ZhuboController->actionView('1')
#2 /var/www/yii-1.1.13.e9e4a0/framework/web/actions/CAction.php(108): ReflectionMethod->invokeArgs(Object(ZhuboController), Array)
#3 /var/www/yii-1.1.13.e9e4a0/framework/web/actions/CInlineAction.php(47): CAction->runWithParamsInternal(Object(ZhuboController), Object(ReflectionMethod), Array)
#4 /var/www/yii-1.1.13.e9e4a0/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /var/www/yii-1.1.13.e9e4a0/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 /var/www/yii-1.1.13.e9e4a0/framework/web/filters/CFilter.php(40): CFilterChain->run()
#7 /var/www/yii-1.1.13.e9e4a0/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 /var/www/yii-1.1.13.e9e4a0/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 /var/www/yii-1.1.13.e9e4a0/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 /var/www/yii-1.1.13.e9e4a0/framework/web/CController.php(291): CFilterChain->run()
#11 /var/www/yii-1.1.13.e9e4a0/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(282): CController->run('view')
#13 /var/www/yii-1.1.13.e9e4a0/framework/web/CWebApplication.php(141): CWebApplication->runController('zhubo/view')
#14 /var/www/yii-1.1.13.e9e4a0/framework/base/CApplication.php(169): CWebApplication->processRequest()
#15 /var/www/webroot/meilizhubo/index.php(14): CApplication->run()
#16 {main}
REQUEST_URI=/index.php?r=zhubo/view&id=1
HTTP_REFERER=http://192.168.216.128:8888/index.php?r=zhubo/index
---
